src/Microsoft.OData.Client/Microsoft.OData.Client.csproj

```diff
@@ -12,8 +12,13 @@
     <DefineConstants>$(DefineConstants);ASSEMBLY_ATTRIBUTE_ON_NETSTANDARD_20</DefineConstants>
   </PropertyGroup>
   
-  <Import Project="..\Build.props" /> 
-
+  <Import Project="..\Build.props" />
+  
+  <PropertyGroup Condition="'$(TreatWarningsAsErrors)' == 'false' or '$(TreatWarningsAsErrors)' == ''">

```

Comment:
Does this not belong in the ../Build.props?

Reply:
We use `Build.props` in our test projects as well, which we don't ship publicly. I can try to consolidate the csproj elements used by our shipped projects into a `.props` file though if you think that makes sense.